pages/_document.js

```diff
@@ -0,0 +1,22 @@
+import Document, { Html, Head, Main, NextScript } from "next/document";
```

Comment:
What is purpose of it?

Reply:
Lighthouse returns this error : The <html> element does not have a [lang] attribute
, so I tried to add lang in inner html tag in _app.js file but it generates new error, so I found this solution and this solution works properly.